packages/common/src/core/schemas/internal/getLocationOptions.ts

```diff
@@ -37,50 +48,33 @@ export async function getLocationOptions(
 // TODO: Maybe move these to outside of core and into respective entities
 
 export async function getContentLocationOptions(
-  entity: ConfigurableEntity,
-  hubRequestOptions: IHubRequestOptions
+  entity: ConfigurableEntity
 ): Promise<IHubLocationOption[]> {
-  const defaultExtent: IExtent = await getGeographicOrgExtent(
-    hubRequestOptions
-  );
-  const location: IHubLocation = entity.location;
-  return (
-    [
-      {
-        label: "{{shared.fields.location.none:translate}}",
-        location: { type: "none" },
+  const defaultExtent: IExtent = getItemExtent(entity.extent);
+  const defaultExtentIsBBox = isBBox(entity.extent);
+  const boundary = entity.boundary;
```

Comment:
is `entity.boundary` a thing? I was expecting that you'd have to reach into `item.properties` to get `boundary`?